components/navigation/MobileNavMenu.tsx

```diff
@@ -41,6 +50,11 @@ export default function MobileNavMenu({ onClickClose = () => {} }: MobileNavMenu
     setOpen(menu);
   }
 
+  const langMap: { [key: string]: string } = {
+    en: 'English',
+    de: 'Deutsch'
+  };
+
```

Comment:
lets move this to `next-i18next.config.js` file. it would keep one source of truth for all i18n configs

Reply:
Done